client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -52,21 +52,34 @@ class StatsPeriodNavigation extends PureComponent {
 
 	handleArrowNext = () => {
 		const { date, moment, period, url, queryParams } = this.props;
-
-		const usedPeriod = 'hour' === period ? 'day' : period;
-		const nextDay = moment( date ).add( 1, usedPeriod ).format( 'YYYY-MM-DD' );
-		const nextDayQuery = qs.stringify( Object.assign( {}, queryParams, { startDate: nextDay } ), {
-			addQueryPrefix: true,
-		} );
+		const tabValue = queryParams.tab;
+		const nextDay = moment( date ).add( 1, period ).format( 'YYYY-MM-DD' );
+		const nextDayQuery = qs.stringify(
+			Object.assign(
+				{},
+				queryParams,
+				{ startDate: nextDay },
+				{ tab: tabValue ? tabValue : 'views' }
```

Comment:
```suggestion

```
Since there could be any page using this `StatsPeriodNavigation` component, indiscriminately appending `tab=views` to another page's URL looks unreasonable. Could we directly determine the query string `tab` via `queryParams`?

Reply:
Hey @annacmc, you could just also keep the original code rather than append the `tabValue` checking for the tab. It worked as what you changed in [22d9722](https://github.com/Automattic/wp-calypso/pull/72040/commits/22d9722287da2a091749993b8328e21b53949348) since the `tab` is already in `queryParams`.

Reply:
You are very right! Thank you